### DISCUSSION TITLE
Set different endpoint to send data to and send clear halt report

### DISCUSSION THREAD

**Discussion by MarcoKonrad** [hathach/tinyusb#2108] [Q&A]
Hello everybody, 

i am trying to send data to a usb hid device. The device seems to recieved the data, since it sends a stall report (Seen with an logic analyzer).
The only two things i havent checked are if the endpoints are wrong and if the device may be halted, so my question is: How would i send a clear halt report and how would i change the endpoint i send data to? I am coding on a raspberry pi pico btw.

Kind regards

**Comment by hathach** [comment]
you cannot change the endpoint address. What is your tusb log ? you should try to analyze why stall occurs
